cog.c

```diff
@@ -75,6 +76,9 @@ static GOptionEntry s_cli_options[] =
     { "config", 'C', 0, G_OPTION_ARG_FILENAME, &s_options.config_file,
         "Path to a configuration file",
         "PATH" },
+    { "bg-color", 'b', 0, G_OPTION_ARG_STRING, &s_options.background_color,
+        "WebView background color to use, by default 'white'. Colors can also be provided with HEX syntax.",
```

Comment:
I would just write `"Background color, as a CSS name or in #RRGGBB or #RRGGBBAA hex syntax (default: white)"`. Because I think that is what `webkit_color_parse` accepts, right?

It is quite clear that the specified color will affect the background behind the Web content without needing to add the word “WebView” here, and also I would like to have the `#RRGGBB or #RRGGBBAA` bit to make it look a bit like an example and be more explicit about how the color has to be specified without having to guess.